dev/src/Command/DocFxCommand.php

```diff
@@ -199,4 +219,37 @@ public static function getPhpDocCommand(string $componentPath, string $outDir):
 
         return $process;
     }
+
+    private function validate(ClassNode $class, OutputInterface $output): bool
+    {
+        $valid = true;
+        $emptyRef = '<options=bold>empty</>';
+        $isGenerated = $class->isProtobufMessageClass() || $class->isProtobufEnumClass() || $class->isServiceClass();
+        foreach (array_merge([$class], $class->getMethods(), $class->getConstants()) as $node) {
+            foreach ($this->getInvalidXrefs($node->getContent()) as $invalidRef) {
+                if (isset(self::$allowedReferenceFailures[$node->getFullname()])
+                    && self::$allowedReferenceFailures[$node->getFullname()] == $invalidRef) {
+                    // these links are inexplicably broken in phpdoc generation, and will require more investigation
```

Comment:
Let's create a ticket for this one.

Reply:
It's a tricky situation because the issue (I think) is in [phpdoc](https://www.phpdoc.org/) itself... but there's no indication AFAIK for why this would happen (they look _identical_ to the other xrefs). It's absolutely baffling.

Before I can file an issue with `phpdoc`, I need to do more investigation to see if I can find out what could be causing it. Because I truly have no clue.

Reply:
Filed an issue in https://github.com/googleapis/google-cloud-php/issues/7377